gsonpath-extensions-compiler/src/main/java/gsonpath/extension/def/intdef/IntDefGsonPathFieldValidator.java

```diff
@@ -8,8 +8,8 @@
  */
 @SuppressWarnings("All")
 @AutoService(GsonPathExtension.class)
-public class AndroidSizeGsonPathFieldValidator extends AndroidSizeGsonPathFieldValidatorImpl {
-    public AndroidSizeGsonPathFieldValidator() {
+public class IntDefGsonPathFieldValidator extends IntDefGsonPathFieldValidatorImpl {
+    public IntDefGsonPathFieldValidator() {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1186)